providers/nodeReleasesProvider.tsx

```diff
@@ -1,12 +1,53 @@
 import { createContext } from 'react';
-import { useFetchNodeReleases } from '../hooks/useFetchNodeReleases';
+import { getNodeReleaseStatus } from '../util/nodeRelease';
+import NodeReleasesJSON from '../public/node-releases-data.json';
 import type { FC, PropsWithChildren } from 'react';
 import type { NodeRelease } from '../types';
 
+type NodeReleaseJSON = {
+  major: number;
+  version: string;
+  codename?: string;
+  currentStart: string;
+  ltsStart?: string;
+  maintenanceStart?: string;
+  endOfLife: string;
+  npm?: string;
+  v8?: string;
+  releaseDate?: string;
+  modules?: string;
+};
+
 export const NodeReleasesContext = createContext<NodeRelease[]>([]);
 
 export const NodeReleasesProvider: FC<PropsWithChildren> = ({ children }) => {
-  const releases = useFetchNodeReleases();
+  const data: NodeReleaseJSON[] = NodeReleasesJSON;
```

Comment:
No need to call this data and define type. Can be called directly on the map as `nodeReleaseData.map`

Reply:
Could you share what you're trying to do? Again as I mentioned you should be able to cast the individual entry inside the `.map()`

Reply:
I misunderstood you. I thought you want to "not define the type" hence why I was confused.